queries.sql

```diff
@@ -8,3 +8,10 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = '1';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= '10.4' AND weight_kg <= '17.3';
+
```

Comment:
- [ ] Good job so far, Kindly add all the queries for transactions in queries.sql. You have correctly written all the queries as per the screenshots attached. Just make sure to write those queries in this file.